pipeline-yaml_5b5744d40428631d7a8940a9.md

```diff
@@ -615,6 +614,53 @@ were also defined on the `task` level.
 	          - echo $VAR_1
 	          - echo $APP_ENV
 
+### matrix
+
+The `martix` property allows you to define one or more environment variables
+with multiple values. After that, it creates multiple jobs by calculating the
+Cartesian product of all the sets of user defined values.
```

Comment:
simplify to "Cartesian product of all sets of values"

(a user is reading this so it doesn't make sense to read user-defined in that perspective)

Reply:
Good point.